documentation/api/wire_format/report_format_v5.markdown

```diff
@@ -5,6 +5,7 @@ canonical: "/puppetdb/latest/api/wire_format/report_format_v5.html"
 ---
 
 [puppetreportformat]: http://docs.puppetlabs.com/puppet/latest/reference/format_report.html
+[reportsv4]: ../query/v4/reports.markdown
```

Comment:
I think the indices in the sidebar for `master` are currently broken, I'm getting 404's for this.